# mandatory item
HttpbinPostBody2 testitem
# optional item
HttpbinPostBody2[] testoptitem
# item array
HttpbinPostBody2[] testitemarray
